cmd/algorand-indexer/main.go

```diff
@@ -125,21 +125,28 @@ func init() {
 
 	if err := viper.ReadInConfig(); err != nil {
 		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
-			// Config file not found; the error message indicates locations where we look
-			fmt.Println(err.Error())
+			// Config file not found. Returns an error string like:
```

Comment:
replace this entire comment block with effectively, "It's okay not to have a config file, maybe everything was set in command line options"